.gitignore

```diff
@@ -1,4 +1,6 @@
-bin/
+!bin/
+bin/*
+!bin/suricata/
```

Comment:
Temporary pending our suricata package. Until then, suricatarunner is part of this repo and so can't be .gitignored.